games-fps/alephone/alephone_marathon-20150907.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="Marathon 1 game assets"
+DESCRIPTION="Alien forces have boarded the interstellar colony ship \
+Marathon. The situation is dire. As a security officer onboard, it is your \
+duty to defend the ship and its crew.
+
+Experience the start of Bungie\'s iconic trilogy with Marathon. This release \
+uses the original Marathon data files for the most authentic experience \
+outside of a classic Mac or emulator."
+HOMEPAGE="https://alephone.lhowon.org/"
+COPYRIGHT="1991-2001 Bungie Software
+	Aleph One Developers"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620/Marathon-20150620-Data.zip"
+SOURCE_DIR="Marathon"
+CHECKSUM_SHA256="774cb40c9bde33a5d23146e82f593ac001edd0ee4d62e5b0445953a95fd44f31"
+ADDITIONAL_FILES="alephone.rdef.in"
+
+ARCHITECTURES="any"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandSuffix=
+fi
+
+PROVIDES="
+	alephone_marathon
+	cmd:marathon$commandSuffix
+	"
+REQUIRES="
+	alephone
+	"
+
+BUILD_REQUIRES="
+	haiku
+	"
+BUILD_PREREQUIRES="
+	cmd:true
+	"
+
+BUILD()
+{
+	true
+}
+
+INSTALL()
+{
+	# Game Assets
+	echo "Copying game data"
+	mkdir -p "$dataDir/AlephOne"
+	cp -r "$sourceDir"/../* "$dataDir/AlephOne"
+
+	# Add Haiku resources
+	local APP_SIGNATURE="application/x-vnd.org.bungie.source.AlephOne"
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local LONG_INFO="$SUMMARY"
+
+	sed \
+		-e "s|@APP_SIGNATURE@|$APP_SIGNATURE|" \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		-e "s|@LONG_INFO@|$LONG_INFO|" \
+		$portDir/additional-files/alephone.rdef.in \
+			> alephone.rdef
+
+	# Startscript
+	echo "Preparing Startscript"
+	mkdir -p $appsDir
+	ALEPHONE="`finddir B_SYSTEM_BIN_DIRECTORY`/alephone"
+	DATA_PATH="`finddir B_SYSTEM_DATA_DIRECTORY`/AlephOne/Marathon"
+
+cat >"$appsDir/Marathon" <<EOF
+#!/bin/sh
+exec $ALEPHONE "$DATA_PATH $1"
```

Comment:
I'm quite sure what we need here is:
```diff
-cat >"$appsDir/Marathon" <<EOF
+	cat >"$appsDir/Marathon" <<EOF 
 #!/bin/sh
-exec $ALEPHONE "$DATA_PATH $1"
+exec $ALEPHONE "$DATA_PATH" "$@"
 EOF
```

(Similar comments for `alephone_marathon2-20150907.recipe` and `alephone_marathoninfinity-20150907.recipe`.)